src/util/webhooks.ts

```diff
@@ -24,7 +24,7 @@ export const emitWebhookProcessedMetrics = (
 	try {
 		// only send logs if time of webhookReceived occurred before the currentTime
 		// and if webhookReceivedTime is not null/undefined
-		if (webhookReceivedTime < currentTime) {
+		if (webhookReceivedTime <= currentTime) {
```

Comment:
I had to change it to `<=` here so it will be easier to use this code in tests when we mock time to a constant value